master/buildbot/process/build.py

```diff
@@ -72,6 +72,9 @@ class Build(properties.PropertiesMixin, WorkerAPICompatMixin):
                         collects our status
     """
 
+    VIRTUAL_BUILDERNAME_PROP = "virtual_buildername"
```

Comment:
The other two properties are of the form `virtual_builder_*` but this one isn't. Seems like that should all have the same prefix.